exonum/src/blockchain/config.rs

```diff
@@ -23,50 +30,84 @@ use storage::StorageValue;
 use crypto::{hash, CryptoHash, Hash, PublicKey};
 use helpers::{Height, Milliseconds};
 
-/// Public keys of a validator.
+/// Public keys of a validator. Each validator has two public keys: the
+/// `consensus_key` is used for internal operations in the consensus process,
+/// while the `service_key` is used in services.
 #[derive(Copy, Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub struct ValidatorKeys {
     /// Consensus key is used for messages related to the consensus algorithm.
-    #[doc(hidden)]
     pub consensus_key: PublicKey,
-    /// Service key is used for services.
+    /// Service key is used for services. This key is used by the configuration
+    /// updater service, the anchoring service, etc.
     pub service_key: PublicKey,
 }
 
-/// Exonum blockchain global configuration.
-/// This configuration must be same for any exonum node in the certain network on given height.
+/// Exonum blockchain global configuration, including validator public keys,
```

Comment:
Do we really need a non-complete description when we have the structure itself with all it contents just below? IMHO it's self-explanatory enough.